package.json

```diff
@@ -46,30 +46,33 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
     "lint:ts": "tsc --noEmit",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "npm-run-all lint:* test:*",
+    "test": "npm-run-all lint test:*",
     "test:node": "qunit node-tests",
     "test:ember": "ember test",
     "test:ember-compatibility": "ember try:each",
     "prepublishOnly": "ember ts:precompile",
     "postpublish": "ember ts:clean"
   },
   "resolutions": {
-    "socket.io": "^4.3.1"
+    "fuse.js": "6.4.6"
```

Comment:
Can you explain this?

Reply:
@scalvert got failure for the "Floating Dependencies" job in https://github.com/ember-a11y/ember-a11y-testing/runs/4608355862?check_suite_focus=true with error

```
error fuse.js@6.5.0: The engine "node" is incompatible with this module. Expected version ">=14". Got "12.22.1"
```

apparently fuse.js@6.5.0 had breaking change yesterday, there is logged issue https://github.com/krisk/Fuse/issues/600

Reply:
previous `socket.io` resolutions seem not needed anymore

Reply:
Makes sense, and thanks for the explanation.

Reply:
@scalvert fuse.js issue was resolved and new patch version released. Rised #352 to remove this resolution